docs/README.md

```diff
@@ -2,6 +2,12 @@
 
 ## Building From Source
 
+Install `mdbook` and then open a new terminal session in order to run the subsequent
+commands
+```sh
```

Comment:
Avoid hard line breaks since they display funny depending on user settings. Wrapping always displays equally correct.

Also needs a newline before codeblocks: https://github.com/DavidAnson/markdownlint/blob/2d19c0685bfa98dce87f493a8fcffbb5939af2c3/doc/Rules.md#md031---fenced-code-blocks-should-be-surrounded-by-blank-lines

````suggestion
Install `mdbook` and then open a new terminal session in order to run the subsequent commands

```sh
````